v0/pipeline/steps/common/traceable-step-info.yaml

```diff
@@ -0,0 +1,140 @@
+title: TraceableStepInfo
+allOf:
+  - $ref: ../../common/step-spec-type.yaml
+  - type: object
+    required:
```

Comment:
- config should be required (see other step infos)
- auth/target shouldn't be required since ingestion won't support it

Reply:
Config isn't needed here.
Since we won't support ingestion, auth/target becomes required